docs/content/nodes/access-api-rate-limits.mdx

```diff
@@ -14,53 +14,55 @@ Please note, these limits only apply to the access nodes hosted by Dapper Labs.
 
 |                 gRPC API                             | Total request per second per client IP        |
 |:-----------------------------------------------------|:---------------------------------------------:|
-| flow.access.AccessAPI/Ping                           |                   2000                        |
-| flow.access.AccessAPI/GetLatestBlockHeader           |                   2000                        |
-| flow.access.AccessAPI/GetBlockHeaderByID             |                   2000                        |
-| flow.access.AccessAPI/GetBlockHeaderByHeight         |                   2000                        |
-| flow.access.AccessAPI/GetLatestBlock                 |                   2000                        |
-| flow.access.AccessAPI/GetBlockByID                   |                   2000                        |
-| flow.access.AccessAPI/GetBlockByHeight               |                   2000                        |
-| flow.access.AccessAPI/GetCollectionByID              |                   2000                        |
-| flow.access.AccessAPI/SendTransaction                |                   2000                        |
-| flow.access.AccessAPI/GetTransaction                 |                   2000                        |
-| flow.access.AccessAPI/GetTransactionResult           |                   2000                        |
-| flow.access.AccessAPI/GetAccount                     |                   2000                        |
-| flow.access.AccessAPI/GetAccountAtLatestBlock        |                   2000                        |
-| flow.access.AccessAPI/GetAccountAtBlockHeight        |                   2000                        |
-| flow.access.AccessAPI/ExecuteScriptAtLatestBlock     |                    40                         |
-| flow.access.AccessAPI/ExecuteScriptAtBlockID         |                    40                         |
-| flow.access.AccessAPI/ExecuteScriptAtBlockHeight     |                    40                         |
-| flow.access.AccessAPI/GetEventsForHeightRange        |                   100                         |
-| flow.access.AccessAPI/GetEventsForBlockIDs           |                   2000                        |
-| flow.access.AccessAPI/GetNetworkParameters           |                   2000                        |
-| flow.access.AccessAPI/GetLatestProtocolStateSnapshot |                   2000                        |
+| flow.access.AccessAPI/Ping                           |                   500                        |
```

Comment:
2000 is still accurate since these rates are applied individually on each of the 4 public nodes. Can multiply all of these by 4, or clearly state somewhere this distinction?